R/options.R

```diff
@@ -16,13 +16,22 @@ get_plot_options <- function() {
     return(as.list(plot.opts))
 }
 
-# all plot mime types. default value for the jupyter.plot_mimetypes option
-plot_mimetypes <- c(
-    'image/png',
-    'application/pdf',
-    'image/svg+xml')
+opt.defaults = list(
+    jupyter.rich_display = TRUE,
+    jupyter.result_mimetypes = c(
+        'text/plain',
+        'text/html',
+        'text/markdown',
+        'text/latex'),
```

Comment:
Sorry, I should have been clearer: I agree with having an option to configure this, but I'd put all the formats we know IPython can display in there by default - I think that just means adding image/png and image/svg+xml to this list.